hathor/simulator/fake_connection.py

```diff
@@ -157,9 +158,27 @@ def run_one_step(self, debug=False, force=False):
 
         return True
 
-    def run_until_complete(self, debug=False, force=False):
+    def _advance_reactors(self, dt: int) -> None:
```

Comment:
It seems this method is not used anymore. Should we remove it?

Reply:
Yes. I'll remove it.

Reply:
Removed.